client/actors-derive/src/lib.rs

```diff
@@ -830,6 +830,91 @@
     result
 }
 
+/// Generate generics for enum and trait declarations from additional generics
+fn generate_additional_generics_for_declaration(
```

Comment:
I find macros useful when there is an example of what the input and output would look like